diff -burNp tml-0.5.1/tml tml-0.5.1-dud/tml
--- tml-0.5.1/tml	2004-07-05 19:28:58.000000000 +0200
+++ tml-0.5.1-dud/tml	2009-03-06 20:40:39.678073276 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/ruby
+#!/usr/bin/ruby
 # $Id$
 #
 # Copyright (C) 2003-2004 TOMITA Masahiro
@@ -8,13 +8,12 @@
 require "getopts"
 require "net/smtp"
 
-$script_dir = File::dirname $0
-conf = $script_dir+"/tml.conf"
+conf = "/etc/mail/tml.conf"
 load conf if File::exist? conf
 
-require $script_dir+"/tml.rb"
-require $script_dir+"/tml-file.rb"
-require $script_dir+"/mail.rb"
+require "tml/tml"
+require "tml/tml-file"
+require "tml/mail"
 
 unless getopts("f:", "domain:") then
   $stderr.puts "invalid usage"
@@ -35,7 +34,7 @@ if not $domain then
 end
 
 if $database == "mysql" then
-  require "#{$script_dir}/tml-mysql.rb"
+  require "tml/tml-mysql"
   $db_class = TML::Mysql
   $spool_class = TML::Mysql
 end
diff -burNp tml-0.5.1/tmladmin tml-0.5.1-dud/tmladmin
--- tml-0.5.1/tmladmin	2004-05-08 16:48:42.000000000 +0200
+++ tml-0.5.1-dud/tmladmin	2009-03-06 20:43:12.911807265 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/ruby
+#!/usr/bin/ruby
 # $Id$
 #
 # Copyright (C) 2003-2004 TOMITA Masahiro
@@ -7,10 +7,9 @@
 
 require "getopts"
 
-$script_dir = File::dirname $0
-require $script_dir+"/tml.rb"
+require "tml/tml"
 
-conf = $script_dir+"/tml.conf"
+conf = "/etc/mail/tml.conf"
 load conf if File::exist? conf
 
 $cmds = {
diff -burNp tml-0.5.1/tmlctl tml-0.5.1-dud/tmlctl
--- tml-0.5.1/tmlctl	2004-05-09 06:07:01.000000000 +0200
+++ tml-0.5.1-dud/tmlctl	2009-03-06 20:44:06.471799721 +0100
@@ -1,4 +1,4 @@
-#!/usr/local/bin/ruby
+#!/usr/bin/ruby
 # $Id$
 #
 # Copyright (C) 2003 TOMITA Masahiro
@@ -8,11 +8,10 @@
 require "getopts"
 require "net/smtp"
 
-$script_dir = File::dirname $0
-require $script_dir+"/tml.rb"
-require $script_dir+"/mail.rb"
+require "tml/tml"
+require "tml/mail"
 
-conf = $script_dir+"/tml.conf"
+conf = "/etc/mail/tml.conf"
 load conf if File::exist? conf
 
 class TmlCtlError < StandardError
diff -burNp tml-0.5.1/tml.rb tml-0.5.1-dud/tml.rb
--- tml-0.5.1/tml.rb	2004-07-04 18:57:31.000000000 +0200
+++ tml-0.5.1-dud/tml.rb	2009-03-06 20:42:15.641813037 +0100
@@ -14,7 +14,7 @@ class TML
   @@parameters = {
     :max_lock_time	=> 30*60,
     :lock_timeout	=> 5*60,
-    :tml_dir		=> "/usr/local/tml",
+    :tml_dir		=> "/usr/lib/tml",
     :var_dir		=> "/var/spool/tml",
     :ml_dir		=> "ml",
     :etc_dir		=> "etc",
@@ -111,10 +111,10 @@ class TML
 
   def self.db_class()
     if defined? $database and $database == "mysql" then
-      require "#{self.dparam(:tml_dir)}/tml-mysql.rb"
+      require "tml/tml-mysql"
       TML::Mysql
     else
-      require "#{self.dparam(:tml_dir)}/tml-file.rb"
+      require "tml/tml-file"
       TML::F
     end
   end
